content/en/Getting started/sign-in.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Sign In to Cobalt"
+linkTitle: "Sign In"
+weight: 9
+description: >
+  Start the pentest process. Sign in to the Cobalt app.
+---
+
+{{% pageinfo %}}
+This page assumes that you've received a welcome email from Cobalt.
+{{% /pageinfo %}}
+
+Even if you haven't yet purchased Cobalt credits, this page (and document)
+can help you visualize how you can set up a pentest with the Cobalt UI.
+
```

Comment:
Do we need to mention anything about sign in when an organization has SAML or Azure SSO, that they will use that provider instead of email+password on Cobalt sign in page?

Reply:
I have to think about this...

Since this is a "Getting Started Guide", I'm defaulting to "no". However, in plans for future docs, I don't have an "Obvious" place for our SAML/3rd Party Sign-in options.

Reply:
Investigating further, it (now) bothers me that we have "Identity & Access" in a different area than 2FA. Gosh.

In any case, I'm now going to expand the note at the end of the [Sign In To Cobalt](https://draft2-getting-started.netlify.app/getting-started/sign-in/) page.

![Screen Shot 2021-09-14 at 12 11 35 PM](https://user-images.githubusercontent.com/84352037/133319635-882311e0-b9de-463a-96a7-0c7aca270aa7.png)